app/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.neenbedankt.android-apt'
 apply plugin: 'hu.supercluster.paperwork'
 apply from: '../code_quality_tools/jacoco.gradle'
 apply plugin: 'com.frogermcs.androiddevmetrics'
+apply from: '../code_quality_tools/quality.gradle'
```

Comment:
Well, it depends, I mean, sometimes you have different modules in the project and you don't want to apply script to all of them.

I'm ok with both variants in case of qualitymatters